src/cmd/run.go

```diff
@@ -477,10 +479,10 @@ func constructExecArgs(container string,
 
 	execArgs = append(execArgs, []string{
 		container,
-		"capsh", "--caps=", "--", "-c", "exec \"$@\"", "/bin/sh",
+		"capsh", "--caps=", "--", "-c",
```

Comment:
This doesn't seem to have anything to do with what the commit message describes.  :)